db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_12_19_044530) do
+ActiveRecord::Schema[6.1].define(version: 2021_12_19_044530) do
```

Comment:
My local version has some updates. We probably need to update this file. Here is the beginning of my diff:
```
--- a/db/schema.rb
+++ b/db/schema.rb
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[6.1].define(version: 2021_12_19_044530) do
-
+ActiveRecord::Schema[7.0].define(version: 2021_12_19_044530) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
@@ -43,17 +42,17 @@ ActiveRecord::Schema[6.1].define(version: 2021_12_19_044530) do
     t.integer "patient_contribution"
     t.integer "naf_pledge"
     t.integer "fund_pledge"
-    t.datetime "fund_pledged_at"
+    t.datetime "fund_pledged_at", precision: nil
...
```